skll/config.py

```diff
@@ -847,15 +847,15 @@ def _parse_config_file(config_path, log_level=logging.INFO):
                          "to use `neg_log_loss` as the objective.")
 
     # set the folds appropriately based on the task:
-    #  (a) if the task is `train` and if an external fold mapping is specified
-    #      then use that mapping for grid search instead of the value
-    #      contained in `grid_search_folds`.
+    #  (a) if the task is `train`/`evaluate`/`predict` and if an external
+    #      fold mapping is specified then use that mapping for grid search
+    #      instead of the value contained in `grid_search_folds`.
     #  (b) if the task is `cross_validate` and an external fold mapping is specified
-    #      then use that mapping for the outer CV loop. Depending on the value of
-    #      `use_folds_file_for_grid_search`, use the fold mapping for the inner
-    #       grid-search loop as well.
+    #      then use that mapping for the outer CV loop and for the inner grid-search
+    #      loop. However, if  `use_folds_file_for_grid_search` is `False, do not
```

Comment:
Nitpick: I think ``` `False``` should be changed to ``` `False` ```. I'm not really sure because this is inline comments. I actually thought values like this should have two pairs of ``` ` ```s, one at each end. We're also using this style (with single ``` ` ```) for other things like ``` `evaluate` ``` (in some cases, though we don't use anything).

Reply:
yeah, we really need to be consistent here. I'll make the change.